x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/action/CreateApiKeyRequestTests.java

```diff
@@ -96,11 +97,22 @@ public void testSerialization() throws IOException {
         }
         request.setRoleDescriptors(descriptorList);
 
+        boolean testV710Bwc = true;// randomBoolean();
```

Comment:
Uncomment the `randomBoolean()`?

Reply:
Aaarg, it should be `randomBoolean` . Thanks for catching Yang!